tests/src/baseservices/compilerservices/RuntimeWrappedException/RuntimeWrappedException.cs

```diff
@@ -34,18 +34,7 @@ public static int Main(string[] args)
 //            var info = new SerializationInfo(typeof(RuntimeWrappedException), new FormatterConverter());
 //            ex.GetObjectData(info,ctx);
 //
-            try
-            {
-                ex.GetObjectData(null,ctx);
-            }
-	    catch (ArgumentNullException ex1)
-            {
-                retVal = 100;
-            }
-            catch (Exception ex1)
-            {
-		retVal = -1;
-            }
+            retVal = 100;
```

Comment:
Can the TODO comment be deleted?

Also, what's the purpose of the above if at this point?  When we get to this point, retVal will be set to 100, regardless of whether it was set to -1 above.